explorer/client/package.json

```diff
@@ -24,9 +31,9 @@
   },
   "main": "app.js",
   "proxy": "http://localhost:8080",
-  "dependencies": {},
-  "devDependencies": {
-    "@babel/preset-typescript": "^7.3.3",
+  "dependencies": {
+    "@babel/core": "7.4.3",
```

Comment:
Any reason why we're pinning this dep but not others?

Reply:
Also, this should be in `devDeps`

Reply:
It's not ejected so fails to boot when not in sync with https://github.com/facebook/create-react-app/blob/v3.0.1/packages/react-scripts/package.json#L31

Reply:
hm, maybe we should leave it unpinned, and instead add a yarn `resolution` key

Reply:
Can you expand on why resolution should be used here?